lightning/src/ln/peer_handler.rs

```diff
@@ -459,6 +459,16 @@ impl<Descriptor: SocketDescriptor, CM: Deref, L: Deref> PeerManager<Descriptor,
 		}
 	}
 
+	/// Append a message to a peer's pending outbound/write buffer, and update the map of peers needing sends accordingly
+	fn enqueue_message<M: Encode + Writeable>(&self, peers_needing_send: &mut HashSet<Descriptor>, peer: &mut Peer, descriptor: Descriptor, message: &M) {
+		let mut encoded_message: Vec<u8> = vec![];
+		wire::write(message, &mut encoded_message);
+
+		log_trace!(self.logger, "Encoding and sending message of type {} to {}", message.type_id().to_string(), log_pubkey!(peer.their_node_id.unwrap()));
```

Comment:
Would it be worth updating this logging given the message is only be enqueued but not sent?

I know the behavior hasn't changed, but the logging accuracy is now dependent on the call site. All the call sites are currently in `do_read_event`, which always calls `do_attempt_write_data`. So in practice the message is sent. (For background, see #456.)

But if `enqueue_message` is called from elsewhere then the logging may not be necessarily accurate.